### DISCUSSION TITLE
soldering shema for esp32 not mini

### DISCUSSION THREAD

**Discussion by johnx102** [hallard/WeMos-TIC#12] [General]
Hi, I just receive my wemos tele info unit, but before order esp32 mini, I want to use my esp32 normal board . Can you explain please what pin on my esp32 I use with the WeMos ? thanks

**Comment by hallard** [comment]
there is no magic all depends on what you want to do, what firmware you use and wire according. 

If you only want to read teleinfo you need to wire :

- GND of your ESP32 to pin labeled GND on Wemos TIC
- 3.3V of your ESP32 to pin labeled 3V3 on Wemos TIC
- GPIO23 of your ESP32 to pin labeled 13 on Wemos TIC

**Comment by johnx102** [comment]
Hi , thanks for answer ;)
I use tasiota firmware with téléinformatiques, and yes just want read data from my linky .
So only 3 wire is enough for that  ? I use esp32 board devkit (bigger than the mini show on you're topic ) https://docs.espressif.com/projects/esp-idf/en/latest/esp32/hw-reference/esp32/get-started-devkitc.html
can you confirm please if for the third is the pin GPIO23 on my device please ?

**Reply by hallard** [reply]
Correct, but in all case it depends on which pin is configured as `TInfo Rx`  in tasmota, in this case it should be  `TInfo RX` set to `GPIO23` in tasmota

Please see some hints on teleinfo with Tasmota
https://tasmota.github.io/docs/Teleinfo/

**Reply by johnx102** [reply]
Ok perfect, thanks I will test that !
